app/views/content/subjects/maths/_article.html.erb

```diff
@@ -138,7 +138,7 @@
     <h2 class="heading-s">Funding for teaching maths</h2>
 
     <p>
-      <a href="/funding-your-training">Funding (money you do not have to pay
+      <a href="/funding-and-support">Funding (money you do not have to pay
```

Comment:
I don't think this is well-worded - I think it probably needs changing to be more like the wording on the engineers page eg 

Scholarships of £xxx or bursaries of £xxx are available for trainee maths teachers. This is money that you do not have to pay back.

Find out about your eligibility for bursaries and scholarships.<link>